### PR TITLE
updated README.md

### PR DESCRIPTION
**Pull request by collinhand**
Updated readme to include github names, a description of the project, and its functionality. Still requires that github usernames of other members be added.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,19 @@
 # Group-4
-Collin Hand, Shean Johnson, Jordan Nelson, Andrew Hoemke
+Members:
+Collin Hand (**leader**) -- aka collinhand
+Shean Johnson -- aka 
+Jordan Nelson -- aka 
+Andrew Hoemke -- aka
```
Comment (AndrewHoemke) [inline README.md:6]: Andrew Hoemke -- aka AndrewHoemke

**Review by AtlasCrusher7 — COMMENTED** [review]
everything looks good to me

File: README.md [hunk 2]
```diff
@@ -1,2 +1,19 @@
 # Group-4
-Collin Hand, Shean Johnson, Jordan Nelson, Andrew Hoemke
+Members:
+Collin Hand (**leader**) -- aka collinhand
+Shean Johnson -- aka
```
Comment (AtlasCrusher7) [inline README.md:4]: aka AtlasCrusher7

File: README.md [hunk 3]
```diff
@@ -1,2 +1,19 @@
 # Group-4
-Collin Hand, Shean Johnson, Jordan Nelson, Andrew Hoemke
+Members:
+Collin Hand (**leader**) -- aka collinhand
+Shean Johnson -- aka 
+Jordan Nelson -- aka
```
Comment (Wisp19) [inline README.md:5]: Wisp19

Comment (collinhand) [reply README.md:5]: Will make the updates.

File: README.md (hunk 2 above)
Comment (collinhand) [reply README.md:4]: Will make the changes.

File: README.md (hunk 1 above)
Comment (collinhand) [reply README.md:6]: Will update the document.

**Review by collinhand — COMMENTED** [review]
Implemented member's changes to document.

**Pull request merged**